website/static/js/projectSettings.js

```diff
@@ -107,7 +107,40 @@ var ProjectSettings = oop.extend(
                     }
                 }
             };
-        }
+        },
+        enableMailingList: function() {
+            var request = $.ajax({
+                type: 'post',
+                dataType: 'json',
+                url: nodeApiUrl + 'mailing_list/'
+            });
+            request.done(function() {
+                window.location.reload();
+            });
+            request.fail($osf.handleJSONError);
+        },
+        disableMailingList: function(nodeType) {
+
+            var message = '<p>Are you sure that you want to disable the mailing list for this ' + nodeType + '?';
```

Comment:
En/Dis-abling mailing lists for a node was done with buttons when I inherited this PR. Proposal changed, wireframes were drafted, and a treebeard that allows admins to manage that for the node and any children was specified as a requirement. This code was dead, and has been removed.